src/systems/filecoin_vm/sysactors/init_actor.id

```diff
@@ -5,6 +5,7 @@ import vmr "github.com/filecoin-project/specs/systems/filecoin_vm/runtime"
 type InitActorState struct {
     // responsible for create new actors
     AddressMap   {addr.Address: addr.ActorID}
+    IDMap        {addr.ActorID: [addr.Address]}
```

Comment:
We went to great effort to remove this, it's not necessary. Given an actor id, it's address is `makeIdAddress(id)`.

The AddressMap above maps _non-ID addresses_ to ids, but there is no reason for the reverse map. An actors ID address is its canonical address.